recipes/simdutf/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "simdutf" %}
+{% set version = "4.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simdutf/simdutf/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e193c4b5912612147c266ea7ebe2108099f3bcab5f0dc34e6289b2a47495ae2f
+
+build:
+  number: 0
```

As you don't have a build script for windows, you need to skip it here.
```suggestion
  number: 0
  skip: true  # [win]
```